python_modules/dagster/dagster/_core/instance/__init__.py

```diff
@@ -3229,4 +3229,4 @@ def get_asset_check_support(self) -> "AssetCheckInstanceSupport":
         )
 
     def backfill_log_storage_enabled(self) -> bool:
-        return False
+        return True
```

Comment:
What does this do?

Reply:
ah it turns on storing backfill logs. I'll revert this before merging so that it's actually pulled from the plus org setting